common/lib/xmodule/xmodule/js/spec/video/video_caption_spec.js

```diff
@@ -158,14 +164,61 @@
                 });
             });
 
+            describe('renderCaptions', function() {
+
+                describe('is rendered', function() {
+                    var KEY = $.ui.keyCode,
+
+                    keyPressEvent = function(key) {
+                        return $.Event('keydown', { keyCode: key });
+                    };
+
+                    it('toggle the captions on control click', function() {
```

Comment:
Nit: 'toggles' not 'toggle' to make the sentence read correctly (and to match the other tests)